tests/Silverback.Integration.Tests/Messaging/Outbound/Routing/OutboxProduceStrategyTests.cs.bak

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) 2020 Sergio Aquilini
```

Comment:
I think that you shouldn't commit this file.

Reply:
Removed